index.js

```diff
@@ -393,8 +394,12 @@ HTTP_RGB.prototype = {
         }
         this.log('Caching Hue as %s ...', level);
         this.cache.hue = level;
-
-        this._setRGB(callback);
+        if (this.cacheUpdated) {
+          this._setRGB(callback);
```

Comment:
Indentation is 4 spaces.  Please follow code consistency.

Reply:
Sorry about that. Fixed